### PR TITLE
Service prototype

### PR DESCRIPTION
**Pull request by Sugar-pack**

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+user_api:
+  bind: users_api:8080
```
Comment (pzabolotniy) [inline config.yml]: **bind** is usually used for the listen action.
as a destination usually is used **addr**

File: main.go [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+		if errLaS != nil && errors.Is(errLaS, http.ErrServerClosed) {
+			logger.Fatal(errLaS)
+		}
+	}()
+
+	<-shutdown
+
+	logger.Info("Shutdown signal received")
+
+	ctx, cancel := context.WithTimeout(context.Background(), shutdownTime)
```
Comment (pzabolotniy) [inline main.go:83]: Why is this for?

Comment (Sugar-pack) [reply main.go:83]: here you set the time that is given for a graceful shutdown

File: config.yml (hunk 1 above)
Comment (Sugar-pack) [reply config.yml]: done

File: config.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+user_api:
+  host: users_api
+  port: 8080
```
Comment (pzabolotniy) [inline config.yml]: you don't need to split destination to **name** and **port**.
you may use **addr** with mixed value, **users_api:8080**

Comment (Sugar-pack) [reply config.yml]: done

**Review by pzabolotniy — APPROVED** [review]

**Pull request merged**